modules/test/playwright/pages/object-web/ObjectDefinitionsPage.ts

```diff
@@ -10,8 +10,11 @@ import {ApplicationsMenuPage} from '../product-navigation-applications-menu/Appl
 export class ObjectDefinitionsPage {
 	readonly addObjectFolderButton: Locator;
 	readonly applicationsMenuPage: ApplicationsMenuPage;
+	readonly confirmFolderNameInput: Locator;
```

Comment:
Hey @rebsilva, I know that this field is called just folder in UI. But what you think in use object prefix here? Maybe this way we can avoid unnecessary misunderstandings due to the next reviewers. 

But I understand why you chose not to use the prefix, so in that case, I leave that decision up to you :)

Reply:
i will change it, thanks!